aspnetcore/fundamentals/minimal-apis/security.md

```diff
@@ -32,7 +32,7 @@ To enable authentication, call [`AddAuthentication`](/dotnet/api/microsoft.exten
 
 :::code language="csharp" source="~/fundamentals/minimal-apis/security/7.0-samples/MinApiAuth/MinApiAuth/Program.cs" id="snippet_1" highlight="2":::
 
-Typically, a specific authentication strategy is used. In the following sample, the app is configured with support for JWT bearer-based authentication.
+Typically, a specific authentication strategy is used. In the following sample, the app is configured with support for JWT bearer-based authentication. This example makes use of the [`Microsoft.AspNetCore.Authentication.JwtBearer`](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.JwtBearer) NuGet package, which isn't provided by the [shared framework](xref:fundamentals/metapackage-app).
```

Comment:
This text feels confusing only because the API referenced isn't actually in the shared framework as the prior clause mentions, it's in a NuGet package.

```suggestion
Typically, a specific authentication strategy is used. In the following sample, the app is configured with support for JWT bearer-based authentication. This example makes use of the APIs available in the [`Microsoft.AspNetCore.Authentication.JwtBearer`](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.JwtBearer) NuGet package.
```

Reply:
I changed it to say "isn't provided," but I defer to your choice of words.

Reply:
Oof, didn't notice! The original verbiage sounds good!

Reply:
It's cool ... I don't think most readers will be confused. If we hear 👂 that readers are confused, we can add the INCLUDE on how to add a package ...

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/includes/package-reference.md

I place that under every instruction in Blazor docs that has the user add a package.